client/Dockerfile

```diff
@@ -0,0 +1,8 @@
+from nginx
+
+maintainer eLife Reviwer Product Team <reviewer-product@elifesciences.org>
+
+copy deployment/nginx.conf /etc/nginx/nginx.conf
+copy dist/ /usr/share/nginx/html
```

Comment:
I assume `dist/` is populated at some point in the build process, so there is the possibility to build the Dockerfile too early and get an empty folder. In general the Dockerfile should always lead to the same result, so that's why that process should be a previous stage of this Dockerfile or a separate image that has been built